Documentation/migration-scripts/python-scripts/push_kafka_messages.py

```
@@ -0,0 +1,293 @@
+            
+
+        log("=== Completed Processing observation-linked dashboard ===")
+        
+        # Flush producer to ensure all messages are sent
+        if producer:
+            producer.flush()
+
+        cursor.close()
+        conn.close()
+
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)